### PR TITLE
DIV-5495 - Add extra sol fields to update contact event

### PR DESCRIPTION
**Pull request by mounikahmcts**
Screenshot 2019-08-13 at 15 07 49

https://tools.hmcts.net/jira/browse/DIV-5495

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseField.json [hunk 1]
```diff
@@ -12430,5 +12430,96 @@
...
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "26/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespSolLinkedEmail",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
```
Comment (hosainnet) [inline definitions/divorce/json/AuthorisationCaseField.json]: @gavinclark do we want caseworkers to be able to modify this email?

Comment (gavinclark) [reply definitions/divorce/json/AuthorisationCaseField.json]: Yes. It was a request from the business. Understand they can't change case IDAM association with out devops intervention but still need to be able to amend email address when necessary.

File: definitions/divorce/json/CaseEventToFields.json [hunk 2]
```diff
@@ -6161,7 +6293,9 @@
     "PageLabel": "Grant Decree Nisi",
     "PageDisplayOrder": 1,
     "FieldShowCondition": "DecreeNisiGranted=\"Yes\" AND DivorceCostsOptionDN != \"endClaim\" AND D8DivorceCostsClaim=\"Y*\"",
-    "ShowSummaryChangeOption": "Yes"
+    "ShowSummaryChangeOption": "Yes",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
+    "RetriesTimeoutURLMidEvent": "120,120"
```
Comment (gavinclark) [inline definitions/divorce/json/CaseEventToFields.json]: Not sure this should be here. /sol-dn-review-petition is used to generate a mini petition link, isn't it?

File: definitions/divorce/json/CaseEventToFields.json [hunk 3]
```diff
@@ -6350,9 +6484,7 @@
     "PageLabel": "Review Acknowledgement of Service",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "No",
-    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
-    "RetriesTimeoutURLMidEvent": "120,120"
+    "ShowSummaryChangeOption": "No"
```
Comment (gavinclark) [inline definitions/divorce/json/CaseEventToFields.json]: This should not have been removed. @hosainnet

File: definitions/divorce/json/CaseEventToFields.json (hunk 2 above)
Comment (gavinclark) [reply definitions/divorce/json/CaseEventToFields.json]: @hosainnet

File: definitions/divorce/json/CaseEventToFields.json [hunk 4]
```diff
@@ -6891,9 +7025,7 @@
     "PageLabel": "Review DN Application",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "No",
-    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
-    "RetriesTimeoutURLMidEvent": "120,120"
+    "ShowSummaryChangeOption": "No"
```
Comment (gavinclark) [inline definitions/divorce/json/CaseEventToFields.json]: @hosainnet again not sure if this should have been removed. if the callback is the one that generates the mini petition preview link I think it is used here

File: definitions/divorce/json/CaseEventToFields.json [hunk 5]
```diff
@@ -7421,9 +7555,7 @@
     "PageLabel": "Review Acknowledgement of Service",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "No",
-    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
-    "RetriesTimeoutURLMidEvent": "120,120"
```
Comment (gavinclark) [inline definitions/divorce/json/CaseEventToFields.json]: @hosainnet again not sure if this should have been removed. if the callback is the one that generates the mini petition preview link I think it is used here

**Pull request merged**